### PR TITLE
set package and namespace to Kentico.Xperience.Disqus.Widget

### PR DESCRIPTION
**Pull request by kentico-ericd**
### Motivation

With the release of the XbK Disqus integration, the NuGet package naming conventions need to be changed. The KX13 package will be named `Kentico.Xperience.Disqus.Widget.KX13` and the XbK package will be named `Kentico.Xperience.Disqus.Widget`.

### Checklist

- [X] Code follows coding conventions held in this repo
- [X] Docs have been updated (if applicable)
- [X] Temporary settings (e.g. variables used during development and testing) have been reverted to defaults

### CODE REVIEW

**Review by KubaOczko — CHANGES_REQUESTED** [review]
The namespace, identifier and other part of code should kept unchanged. We want to just change the nuget package name.

**Review by kentico-ericd — COMMENTED** [review]
Reverted namespace and identifier changes

File: src/Kentico.Xperience.Disqus.csproj [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
-
-	<ItemGroup>
-		<PackageReference Include="Kentico.Xperience.WebApp" Version="22.0.3" />
-	</ItemGroup>
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk.Razor">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp6</TargetFramework>
```
[the remark this answers is not in the document]
Comment (kentico-ericd) [reply src/Kentico.Xperience.Disqus.csproj:4]: done

File: src/Kentico.Xperience.Disqus.csproj [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+		<PackageTags>kentico xperience mvc core disqus comments</PackageTags>
+		<Description>A page builder widget which enables Disqus comments on your Xperience by Kentico website.</Description>
+		<RepositoryUrl>https://github.com/Kentico/xperience-disqus/tree/xbk</RepositoryUrl>
+		<RepositoryType>git</RepositoryType>
+		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+		<GenerateDocumentationFile>true</GenerateDocumentationFile>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<None Include="..\img\icon.png" Pack="true" PackagePath="" />
```
[the remark this answers is not in the document]
Comment (kentico-ericd) [reply src/Kentico.Xperience.Disqus.csproj:25]: the icon has been updated

**Review by KubaOczko — APPROVED** [review]

**Pull request merged**